### DISCUSSION TITLE
Lammps not installing in Colab Tutorial

### DISCUSSION THREAD

**Discussion by ingcoder** [mir-group/nequip#320] [Q&A] [answered]
Hi There, 
I am trying to run the NequIP Colab Tutorial but the installation of Lammps (second cell) stalls after 10%. See image below. Does anyone experience the same issue and knows how to fix this?
<img width="1078" alt="Screenshot 2023-03-21 at 7 48 23 PM" src="https://user-images.githubusercontent.com/47470591/226790030-7c649bc4-7983-46cd-b073-81cf0e4b0518.png">

**Comment by simonbatzner** [comment, marked as answer]
Thanks for bringing this to our attention, just updated the tutorial. Let me know if it fixes it.

**Reply by ingcoder** [reply]
Thank you for your quick response! It's fixed! One more question. Is there any instructions on running the code on an Apple M1 machine? I can't install the mkl packages on an M1. When I compile the code on my local machine i get the same error as above but pointing to force.cpp.o instead of pair_nequip.cpp.0 . Any idea?
<img width="328" alt="Screenshot 2023-03-22 at 10 53 04 AM" src="https://user-images.githubusercontent.com/47470591/226995576-deb83499-ad95-4477-a10f-9eeed9e5d288.png">

**Reply by simonbatzner** [reply]
Hi @ingcoder, we currently don't support the M1 architecture in any way, since it's not a priority for scientific workloads at the moment. If you can make it work, we'd be happy to accept it.

**Reply by ingcoder** [reply]
Thank you! I will figure it out then on my own and share.

**Reply by Linux-cpp-lisp** [reply]
Following up on the M1 point:

In general the Python/PyTorch side of our packages works fine on M1 (I use it all the time) when running on the `cpu` device.  The `mps` backend (for the M1 GPU) still doesn't support enough ops in PyTorch to run our models on the M1 GPU, however.

We have no support for M1 with LAMMPS and I'm not even sure if LAMMPS supports that;  I would encourage you to also consider building and running this through the x86 compatibility layer or, even better, finding a Linux server to run things on. (You'll want a more powerful machine for serious MD under any circumstances.)

**Reply by ingcoder** [reply]
Thank you that helps. I have been trying for hours now to get it to run, but will move it to a Linux server on GCP instead.
